q2_feature_table/_rename.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import warnings
+
+import biom
+import qiime2
+
+
+def _generate_new_names(old_ids, rename, strict, verbose=False):
+    """
+    Checks the list of samples ot be renamed
+    """
+    # all old ids need to be unique
+    if (rename.value_counts() > 1).any():
+        raise ValueError('All new ids must be unique.\n'
+                         'Try qiime feature-table group if you want '
```

Comment:
I like where you're going with this, but the formatting of the command might be confusing, since it is hardcoded as a q2cli invocation. I think in a case like this you can get away with just referring to the "method `group` on this plugin", or something along those lines.

Reply:
I updated as suggested, although without the formatting. I don't think my CLI interpreter can handle the back ticks. Can you do a **bold** in command line arguments easily?